### PR TITLE
Remove underline in dropdown button & add minor changes in its design

### PR DESCRIPTION
**Pull request by kubaolszewski**

### CODE REVIEW

File: lib/app/features/competiton/competition_widgets/competition_search_bar.dart [hunk 1]
```diff
@@ -6,56 +6,74 @@ import 'package:score_live/app/features/competiton/cubit/competition_cubit.dart'
 import 'package:score_live/core/applocalization_context.dart';
 import 'package:score_live/presentation/constants/app_colors.dart';
 import 'package:score_live/presentation/constants/app_const_variables.dart';
+import 'package:score_live/presentation/constants/text_styles.dart';
 
 class CompetitionSearchBar extends StatelessWidget {
   CompetitionSearchBar({super.key});
 
+  final List<String> searchTypes = <String>['Team', 'League (name)'];
```
Comment (krzychudz) [inline lib/app/features/competiton/competition_widgets/competition_search_bar.dart]: What is the purpose of this?

Comment (kubaolszewski) [reply lib/app/features/competiton/competition_widgets/competition_search_bar.dart]: this list is provided as a dropdown button options later in the code, I didn't want them to be coded inside the widget so I extracted whole list outside of build method

Comment (krzychudz) [reply lib/app/features/competiton/competition_widgets/competition_search_bar.dart]: So If the list does not change than I would keep such a list in separate class with constant values or something else but not in the widget itself

Comment (kubaolszewski) [reply lib/app/features/competiton/competition_widgets/competition_search_bar.dart]: good point

**Review by krzychudz — APPROVED** [review]

**Pull request merged**